docs/getting_started.md

```diff
@@ -1,187 +1,268 @@
-Defualt endpoint (http://localhost:3000/schema/?format=corejson)
+result:
 
-```bash
-export METEOROID_SCHEMA_ENDPOINT=http://host:port/schema/?format=corejson
+```
++---------------------+-------------------------------------------------------------+
+| Field               | Value                                                       |
++---------------------+-------------------------------------------------------------+
+| id                  | 1                                                           |
+| code                | def main(dict):                                             |
+|                     |     temperature = dict['data'][0]['temperature']['value']   |
+|                     |     threshold = int(dict['threshold'])                      |
+|                     |     message = "No problem."                                 |
+|                     |     if temperature > threshold:                             |
+|                     |         message = "Dangerous because it’s too hot!"         |
+|                     |     print(temperature, message)                             |
+|                     |     return {"temperature": temperature, "message": message} |
+|                     |                                                             |
+| language            | python:3                                                    |
+| binary              | False                                                       |
+| main                |                                                             |
+| version             | 0.0.1                                                       |
+| parameters          | [{'key': 'threshold', 'value': '35'}]                       |
+| fiware_service      |                                                             |
+| fiware_service_path | /                                                           |
+| name                | Thermometer                                                 |
++---------------------+-------------------------------------------------------------+
 ```
 
----
+### Create Endpoint
 
-## Usage
+Create a endpoint with the name hello.
+For function ID, specify the ID of the Function created in the previous step.
 
-#### Create a Python script
+`meteoroid endpoint create [basepath(name)] [path] [method] [function ID]`
 
-Create a Python script to register as Function.
-```python
-def main(arg):
-    return {"test": "test"}
+```bash
+meteoroid endpoint create room /thermometer post 1
 ```
 
-### Create Function
+result:
+
+```bash
+{'id': 1, 'url': 'http://192.168.128.100:9090/api/23bc46b1-71f6-4ed5-8c54-816aa4f8c502/room/thermometer', 'fiware_service': '', 'fiware_service_path': '/', 'name': 'room', 'path': '/thermometer', 'method': 'post', 'function': 1}
+```
 
-Create a function with the name function1.
-You can pass parameters to Function with the param option.
+### Invoke the function using curl
+Invoke using the URL obtained when created endpoint.
```

Comment:
I will fix this line such as following sentence.

Invoke the function with the URL obtained when created endpoint.

Reply:
Done